docs/cri/config.md

```diff
@@ -77,6 +77,16 @@ version = 2
   # limit.
   max_container_log_line_size = 16384
 
+  # container_log_limit is the maximum number of log lines allowed to write to disk per second.
+  # If more lines than specified in container_log_limit= are logged by a container in 1 second,
+  # all further log lines within this second are dropped until this second is over.
+  # Non-positive value means no limit.
+  container_log_limit = 0
```

Comment:
Dropping seems fine, I wonder if we should have another option to set the rate limiting strategy to drop vs block.

Reply:
If we block it when writing logs to disk, FIFO will block too, and then the application will be throttled